src/sql-parser/src/parser.rs

```diff
@@ -2057,6 +2081,42 @@ impl<'a> Parser<'a> {
         })
     }
 
+    fn parse_avro_doc_on_option_name(&mut self) -> Result<DocOnIdentifier<Raw>, ParserError> {
+        match self.expect_one_of_keywords(&[TYPE, COLUMN])? {
+            TYPE => Ok(DocOnIdentifier::Type(self.parse_raw_name()?)),
+            COLUMN => {
+                let (item_name, column_name) = self.parse_column_name()?;
+                Ok(DocOnIdentifier::Column {
+                    item_name,
+                    column_name,
+                })
+            }
+            _ => unreachable!(),
+        }
+    }
+
+    fn parse_column_name(&mut self) -> Result<(RawItemName, Ident), ParserError> {
+        let start = self.peek_pos();
+        let mut item_name = self.parse_raw_name()?;
+        let column_name = match &mut item_name {
+            RawItemName::Name(UnresolvedItemName(identifiers)) => {
+                if identifiers.len() < 2 {
+                    return Err(ParserError::new(
+                        start,
+                        "need to specify an object and a column",
+                    ));
```

Comment:
Uncovered

Reply:
They should be but it ran into a timeout and thus no coverage file was created. Sorry for the false alarm.

Reply:
No worries 👍